src/components/home/elements/BurnToken.vue

```diff
@@ -591,24 +596,23 @@ export default {
             if (BigNumber(amount).isEqualTo(0)) {
                 return void 0
             }
-            return !BigNumber(amount).isNaN() && !this.isWrongFormat(amount) && !this.isNegative(amount) && !this.isTokenInsufficient(amount)
+            return this.checkPrecision(amount) && this.isNumFormatValid(amount) && !this.isTokenInsufficient(amount) && !this.isInsufficient() && !this.isNegative(amount)
```

Comment:
why `isInsufficient` should be included here

Reply:
This method is final check.If you don't have enough VSYS, you can't execute destroy token function.It is same as what is in IssueToken.vue module, but it was forgot to add, so it is included now.

Reply:
the method is called `isAmountValid` which check the validation of variable `amount` (Token Balance).
however, `isInsufficient` checked the balance of VSYS.
they are different concepts.

if the two concepts merged, what the usage of line 64-67? since line 54 checked the insufficience before.

Reply:
yes, they are different concepts.We put _isInsufficient_ in isAmountValid method in order to show error message in the same place. The error message (line 64-67) will show only when it is checked in line 54 first.
<img width="1128" alt="WechatIMG273" src="https://user-images.githubusercontent.com/32302909/58608804-cec35c80-82d7-11e9-831e-0afa0ad02f6b.png">

Reply:
from the image, a “x” shows that the amount is invalid, but the result is insufficient VSYS. It is a logical confused result! A separation implementation is needed.